schemas/settings.py

```diff
@@ -63,7 +63,7 @@
         },
         {
             'name': 'sys0',
-            'descr': 'System for testing check dependencies',
+            'descr': 'System for test dependencies unit tests',
```

Comment:
```suggestion
            'descr': 'System for checking test dependencies',
```

Reply:
I initially changed that to match the description in `unittests/resources/settings.py`. So I will also update that file with the recommended description.